PhysicsTools/NanoAOD/plugins/BToKstllProducer.cc

```diff
@@ -1241,7 +1265,8 @@ void BToKstllProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup
 
```

Comment:
in line 1184 as well add

if(isLeading || isSubleading)
checkLeptonsDuplicateTriplet[muonTrg_index][dummyPair] = loop_index;